### ISSUE TITLE
E3690 false positive when specify EngineVersion without minor for aurora postgresql serverless v2

### ISSUE THREAD

**Issue by georgej-github** [aws-cloudformation/cfn-lint#3976]
### CloudFormation Lint Version

1.26.1

### What operating system are you using?

Mac, Amazon Linux 2023

### Describe the bug

I get

```
E3690 13 is not one of ['11.21', '11.9', '12.15', '12.16', '12.17', '12.18', '12.19', '12.20', '12.22', '12.9', '13.11', '13.12', '13.13', '13.14', '13.15', '13.16', '13.18', '13.9', '14.10', '14.11', '14.12', '14.13', '14.15', '14.6', '14.8', '14.9', '15.10', '15.3', '15.4', '15.5', '15.6', '15.7', '15.8', '16.1', '16.2', '16.3', '16.4', '16.4-limitless', '16.6', '16.6-limitless']
roles/deploy/files/test.yaml:66:7
```
on the reproduction template

### Expected behavior

The attached reproduction template snippet works with this lint rule ignored (can successfully deploy it)

### Reproduction template

The following snippet works with this lint rule ignored
```
Resources:
  AuroraV2Cluster:
    Type: AWS::RDS::DBCluster
    Properties:
      Engine: aurora-postgresql
      EngineVersion: 13
      DatabaseName: test
      MasterUsername: test
      MasterUserPassword: test
 ```
